english/trapezoid/trapezoid.md

```diff
@@ -5,3 +5,11 @@ You draw a perfect trapezoid on the blackboard
 The finished trapezoid starts to glow and a hole appears
 
 Mystified, you enter the trapezoid
+
+This trapezoid world gives every traveler a square to dance on. 
+
+The right type of dance will crack the square open. 
+For example [Square dance](https://en.wikipedia.org/wiki/Square_dance)
+
+However, you could essentially be pick the wrong type of dance which in turn will land you
+in a world with where people only spoke like Deepak Chopra where the mind is purely a consonant in the existence of all there is to be. Click [here](train-ride/train-ride.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.